The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-02-14 13:03:46.529 UTC [49] LOG:  listening on IPv6 address "::1", port 5432
2025-02-14 13:03:46.529 UTC [49] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-14 13:03:46.531 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:03:46.537 UTC [50] LOG:  database system was shut down at 2025-02-14 13:03:46 UTC
2025-02-14 13:03:46.546 UTC [49] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE

CREATE ROLE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 13:03:46.914 UTC [49] LOG:  received fast shutdown request
2025-02-14 13:03:46.915 UTC [49] LOG:  aborting any active transactions
2025-02-14 13:03:46.915 UTC [49] LOG:  worker process: logical replication launcher (PID 56) exited with exit code 1
2025-02-14 13:03:46.916 UTC [51] LOG:  shutting down
2025-02-14 13:03:46.924 UTC [49] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 13:03:47.022 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 13:03:47.022 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 13:03:47.024 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 13:03:47.031 UTC [76] LOG:  database system was shut down at 2025-02-14 13:03:46 UTC
2025-02-14 13:03:47.039 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 13:03:54.242 UTC [91] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 13:03:54.242 UTC [91] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 13:03:54.574 UTC [91] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:03:54.574 UTC [91] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_b5a64c7e-9802-411f-a8ff-3eca17c763ee" WITH (FORCE)
2025-02-14 13:03:55.052 UTC [91] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:03:55.052 UTC [91] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_e817f054-ef9c-4c26-a15c-d1d1419974d4" WITH (FORCE)
2025-02-14 13:03:55.454 UTC [91] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:03:55.454 UTC [91] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_07c648b5-38ea-48f1-80ce-e32a9954678a" WITH (FORCE)
2025-02-14 13:03:55.953 UTC [91] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 13:03:55.953 UTC [91] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_e4d038d6-d572-44e5-ba29-b9e1e807a6f3" WITH (FORCE)
